libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/ContextualizedLogger.scala

```diff
@@ -28,6 +28,10 @@ object ContextualizedLogger {
   private[logging] def createFor(name: String): ContextualizedLogger =
     createFor(LoggerFactory.getLogger(name))
 
+  // Special cases for Daml trace and Daml Warning
+  def damlTraceLog: ContextualizedLogger = createFor("daml.tracelog")
+  def damlWarnings: ContextualizedLogger = createFor("daml.warnings")
```

Comment:
I don't think that special cases that are specific to the Daml engine belong in a library used everywhere else. Can you move these to `//daml-lf`?

Reply:
Those are special, it the sens they come from Daml (those are log and trace the Daml user writes).
I am fine to change it  to `daml-lf`. 
@cocreature what is your opinion ?

Reply:
let’s move it.